lib/model.js

```diff
@@ -508,41 +508,41 @@ module.exports = function(registry) {
     var findByIdFunc = this.prototype['__findById__' + relationName];
     define('__findById__' + relationName, {
       isStatic: false,
-      http: {verb: 'get', path: '/' + pathName + '/:fk'},
-      accepts: {arg: 'fk', type: 'any',
+      http: { verb: 'get', path: '/' + pathName + '/:fk' },
+      accepts: { arg: 'fk', type: 'any',
         description: 'Foreign key for ' + relationName, required: true,
-        http: {source: 'path'}},
+        http: { source: 'path' }},
       description: 'Find a related item by id for ' + relationName + '.',
       accessType: 'READ',
-      returns: {arg: 'result', type: toModelName, root: true},
-      rest: {after: convertNullToNotFoundError.bind(null, toModelName)}
+      returns: { arg: 'result', type: toModelName, root: true },
+      rest: { after: convertNullToNotFoundError.bind(null, toModelName) },
     }, findByIdFunc);
 
     var destroyByIdFunc = this.prototype['__destroyById__' + relationName];
     define('__destroyById__' + relationName, {
       isStatic: false,
-      http: {verb: 'delete', path: '/' + pathName + '/:fk'},
-      accepts: {arg: 'fk', type: 'any',
+      http: { verb: 'delete', path: '/' + pathName + '/:fk' },
+      accepts: { arg: 'fk', type: 'any',
         description: 'Foreign key for ' + relationName, required: true,
-        http: {source: 'path'}},
+        http: { source: 'path' }},
       description: 'Delete a related item by id for ' + relationName + '.',
       accessType: 'WRITE',
-      returns: []
+      returns: [],
     }, destroyByIdFunc);
 
     var updateByIdFunc = this.prototype['__updateById__' + relationName];
     define('__updateById__' + relationName, {
       isStatic: false,
-      http: {verb: 'put', path: '/' + pathName + '/:fk'},
+      http: { verb: 'put', path: '/' + pathName + '/:fk' },
       accepts: [
-        {arg: 'fk', type: 'any',
+        { arg: 'fk', type: 'any',
           description: 'Foreign key for ' + relationName, required: true,
-          http: {source: 'path'}},
-        {arg: 'data', type: toModelName, http: {source: 'body'}}
+          http: { source: 'path' }},
+        { arg: 'data', type: toModelName, http: { source: 'body' }},
```

Comment:
I don't mind either way, I see the custom configs now TY. I prefer less typing lol, so let's keep it the way have it now. It just confused me because we use a space in front of objects but no corresponding space at the end.